### DISCUSSION TITLE
Circles that are in the same or very similar spot get clicked at the same time

### DISCUSSION THREAD

**Discussion by supreme-gay** [ppy/osu#19334] [Q&A]
### Type

Game behaviour

### Bug description

During gameplay if two beats are on top of each other, they will click at the same time, ruining accuracy and general playability.

### Version

2022.723.0

### Logs

[runtime.log](https://github.com/ppy/osu/files/9173960/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9173961/updater.log)
[database.log](https://github.com/ppy/osu/files/9173962/database.log)
[network.log](https://github.com/ppy/osu/files/9173963/network.log)
[performance.log](https://github.com/ppy/osu/files/9173964/performance.log)

**Comment by bdach** [comment]
Please upload a video of this and link the map where you found this issue.

**Comment by peppy** [comment]
Based on previous occurrences of this, you likely have your key bindings setup wrong, and have the same key bound twice.
